examples/returns-from-sap/main.bal

```diff
@@ -0,0 +1,91 @@
+import ballerina/io;
+import ballerina/time;
+import ballerinax/googleapis.gmail as gmail;
+import osa/api_customer_returns_delivery_srv_0002 as returnDel;
+
+configurable string hostname = ?;
+configurable string username = ?;
+configurable string password = ?;
+
+configurable string refreshToken = ?;
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+
+configurable string recipientAddress = ?;
+configurable string fromAddress = ?;
```

Comment:
Shall we group them togeather based on the client?

Reply:
Updated in [Add reviewed changes and documentation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/15/commits/918d0df8497a939d2833b9ed1b4509b83f1f89c0)